website/mailing_list/model.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+import datetime
+
+from modularodm import fields, StoredObject
+
+from framework.mongo import ObjectId
+from framework.mongo.utils import unique_on
+
+@unique_on(['destination_node', '_id'])
```

Comment:
Does this actually do anything?
`_id` is already unique due to being a primary key.
This will result in no constraint being applied as `_id` will always be a newly generated unique value.